lib/travis/build/script/perl6.rb

```diff
@@ -0,0 +1,45 @@
+# Maintained by:
+# Paul Cochrane    @paultcochrane   paul@liekut.de
+
+module Travis
+  module Build
+    class Script
+      class Perl6 < Script
+        DEFAULTS = {
+          perl6: 'latest'
+        }
+
+        def configure
+          super
+
+          sh.echo ''
+          sh.echo 'Perl6 support for Travis-CI is community maintained.', ansi: :red
+          sh.echo 'Please open any issues at https://github.com/travis-ci/travis-ci/issues/new', ansi: :red
+
+          sh.echo 'Installing Rakudo (MoarVM)', ansi: :yellow
+          sh.cmd 'git clone https://github.com/rakudo/rakudo.git'
+          sh.cmd 'cd rakudo'
+          sh.cmd 'sudo perl Configure.pl --backends=moar --gen-nqp --gen-moar --prefix=/usr'
+          sh.cmd 'sudo make install'
```

Comment:
Cloning the Rakudo repo and then building and installing MoarVM, NQP and finally Rakudo took 344s on the Travis infrastructure in a speed test today.  Using `rakudobrew` for the same process took 394s.  The variation could be purely statistical.  Thus one could say that a full Rakudo build and installation takes about 6 minutes.